arm-compute/2016-03-30/swagger/compute.json

```diff
@@ -4561,6 +4681,14 @@
         }
       },
       "x-ms-azure-resource": true
+    },
+    "GenericOkResponse": {
+      "properties": {
+        "status": {
```

Comment:
This model also contains other properties like
- startTime
- endTime
- name

Why are they not modeled over here?